input/pagecontent/changes.md

```diff
@@ -10,6 +10,7 @@ This change log documents the significant updates and resolutions implemented fr
 - Added Security and Privacy page [FHIR-45067](https://jira.hl7.org/browse/FHIR-45067).
 - Removed Must Support from Location.description in AU Core Location [FHIR-45029](https://jira.hl7.org/browse/FHIR-45029).
 - Made the following changes to AU Core Medication:
+  - removal of Must Support from Medication.manufacturer FHIR-45130.
```

Comment:
Missing hyperlink to JIRA issue. 
Also has a full stop where others don't, making it inconsistent with other entries (noting a lack of the change log formatting conventions).

Reply:
fixed both.